.github/workflows/cypress.yaml

```diff
@@ -22,20 +22,28 @@ jobs:
         run: |
           pip install poetry
           poetry install
-      - name: set opscenter config
+      - name: set config
         env:
           SNOWFLAKE_ACCOUNT: ${{ secrets.SNOWFLAKE_ACCOUNT }}
           SNOWFLAKE_USERNAME: ${{ secrets.SNOWFLAKE_USERNAME }}
           SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
         run: |
           mkdir $HOME/.snowsql
           envsubst < deploy/opscenter.config > $HOME/.snowsql/config
-      - name: set opscenter env
+      - name: set env
         run: |
-          echo "OPSCENTER_PACKAGE=PACKAGE_DEV" >> "$GITHUB_ENV"
-          echo "OPSCENTER_DATABASE=\"DATABASE_OPSCENTER_DEV\"" >> "$GITHUB_ENV"
-          echo "OPSCENTER_APP=APP_OPSCENTER_DEV" >> "$GITHUB_ENV"
-      - name: run opscenter streamlit
+          suffix=$(echo ${GITHUB_HEAD_REF} | sed "s/\./_/g" | sed "s/-/_/g" | sed "s\/\_\g")_${GITHUB_RUN_ID}
+          echo "OPSCENTER_PACKAGE=PACKAGE_${suffix}" >> "$GITHUB_ENV"
```

Comment:
we prob want to chang ethis to 
`echo "OPSCENTER_PACKAGE=PACKAGE_CYPRESS_${suffix}" >> "$GITHUB_ENV"`
similar to the ones below
otherwise this job may step on the other CI job (which uses the same suffix for its deploy)

Reply:
Is `PACKAGE_CYPRESS_${suffix}` a value that exists?

Reply:
thats the value. the key is OPSCENTER_PACKAGE. Or am I misunderstanding something